backend/test_message_persistence_and_streaming.py

```
@@ -0,0 +1,403 @@
+                print("   ✅ First message sent successfully")
+                messages_sent.append("I want to build a todo app")
+                
+                # Read response to completion
+                async for line in response.content:
+                    line_str = line.decode('utf-8').strip()
+                    if line_str.startswith('data: '):
+                        try:
+                            data = json.loads(line_str[6:])
+                            if data.get('type') == 'complete':
+                                break
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)